harness/tests/src/main/kotlin/godot/tests/Invocation.kt

```diff
@@ -139,8 +189,124 @@ class Invocation : Spatial() {
 
     override fun _onDestroy() {
         invocation.free()
+        navMeshes.clear()
+        navMeshesDictionary.clear()
+        anyToAnyDictionary.clear()
+        nullableArray.clear()
+        nullableDictionary.clear()
+        testArrayAny.clear()
```

Comment:
Why are these needed? Shouldn't they be cleared by the GC?

Reply:
Forgot to remove them, was during debugging